--- origsrc/ocaml-gavl-0.1.4/configure	2011-05-12 15:53:58.000000000 -0500
+++ src/ocaml-gavl-0.1.4/configure	2011-12-15 17:37:37.670058700 -0600
@@ -4300,7 +4300,7 @@ CAMLLIBPATH=$OCAMLLIB
 ac_config_headers="$ac_config_headers config.h:config.h.in"
 
 # Include it
-CFLAGS="$CFLAGS -I`pwd`/`dirname $0`"
+CFLAGS="$CFLAGS -I`pwd`"
 
 for ac_func in memalign
 do :
--- origsrc/ocaml-gavl-0.1.4/configure.ac	2011-05-12 15:53:58.000000000 -0500
+++ src/ocaml-gavl-0.1.4/configure.ac	2011-12-15 17:36:51.921442000 -0600
@@ -18,7 +18,7 @@ AC_CHECK_OCAML_COMPILERS()
 # Include a config.h
 AC_CONFIG_HEADERS([config.h:config.h.in])
 # Include it
-CFLAGS="$CFLAGS -I`pwd`/`dirname $0`"
+CFLAGS="$CFLAGS -I`pwd`"
 
 AC_CHECK_FUNCS([memalign])
 
