--- origsrc/ocaml-gavl-0.1.0/src/Makefile.in	2009-02-17 12:40:52.000000000 -0600
+++ src/ocaml-gavl-0.1.0/src/Makefile.in	2009-03-04 17:18:04.726625000 -0600
@@ -27,7 +27,7 @@ SOURCES = gavl_stubs.c gavl.ml gavl.mli
 RESULT = gavl
 OCAMLDOCFLAGS = -stars
 LIBINSTALL_FILES = $(wildcard *.mli *.cmi *.cma *.cmxa *.cmx *.a *.so)
-ACLIBS = @LIBS@ @gavl_LIBS@
+ACLIBS = bigarray @LIBS@ @gavl_LIBS@
 LDFLAGS = @LDFLAGS@ @gavl_LDFLAGS@
 CLIBS = $(ACLIBS:-l%=%)
 CC = @CC@
